### PR TITLE
Span conversions + consistency changes

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
So, the branch name is a little indicative of how I fell down this rabbit hole.
I was playing around with light tiles (those ones that light up different colors) when I noticed one of the chat messages wasn't working. As in, nothing at all was showing up, just x4 x5 x6 etc.
In the code, it was ```to_chat(user, "<span class='warning> ....")``` < notice the typo that caused it to not show at all
I corrected it, then switched it to the new helper for span_warning.
Then I got a little curious about the occurrence of traditional spans, pressed FIND, and here we are.

### This PR aims to do two things:
1. Convert spans to the helper functions for easier management
2. Make user feedback a little more consistent.

### My current train of thought is this:
1. notices are for things happening
2. boldnotice is for events that happen with emphasis / ends in !
3. info is for info text
4. warning is when something goes wrong
5. danger is when the user could be injured/ is lightly injured
6. userdanger is when the user was injured/killed

Adds in some new spans to match the theme:

this is binary notice (OBEY THESE LAWS), binary announce (bottom)

## Why It's Good For The Game
1. consistency
2. readability
3. fixed bugs

## Changelog

:cl:
fix: Fixed many chat alerts that were not displaying properly / at all because of typos
code: Some alert types have been changed for consistency, flavor.
add: Added a new alert type for borg laws and directives.
/:cl:

### CODE REVIEW

File: code/datums/brain_damage/hypnosis.dm [hunk 1]
```diff
@@ -25,14 +25,14 @@
 /datum/brain_trauma/hypnosis/on_gain()
 	message_admins("[ADMIN_LOOKUPFLW(owner)] was hypnotized with the phrase '[hypnotic_phrase]'.")
 	log_game("[key_name(owner)] was hypnotized with the phrase '[hypnotic_phrase]'.")
-	to_chat(owner, "<span class='reallybig hypnophrase'>[hypnotic_phrase]</span>")
-	to_chat(owner, "<span class='notice'>[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
+	to_chat(owner, span_hypnophrase("[hypnotic_phrase]"))
```
Comment (MrMelbert) [inline code/datums/brain_damage/hypnosis.dm:28]: This hypnophrase won't be bigtext like the original span did

File: code/datums/brain_damage/mild.dm [hunk 2]
```diff
@@ -175,7 +175,7 @@
 /datum/brain_trauma/mild/nervous_cough/on_life(delta_time, times_fired)
 	if(DT_PROB(6, delta_time) && !HAS_TRAIT(owner, TRAIT_SOOTHED_THROAT))
 		if(prob(5))
-			to_chat(owner, "<span notice='warning'>[pick("You have a coughing fit!", "You can't stop coughing!")]</span>")
```
Comment (jlsnow301) [inline code/datums/brain_damage/mild.dm:178]: ```class='warning'```

File: code/datums/brain_damage/hypnosis.dm [hunk 3]
```diff
@@ -25,14 +25,14 @@
...
+	to_chat(owner, span_notice("[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
 												"Your head hurts, but this is all you can think of. It must be vitally important.",\
 												"You feel a part of your mind repeating this over and over. You need to follow these words.",\
 												"Something about this sounds... right, for some reason. You feel like you should follow these words.",\
-												"These words keep echoing in your mind. You find yourself completely fascinated by them.")]</span>")
-	to_chat(owner, "<span class='boldwarning'>You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so,\
-										as long as you act like the words are your highest priority.</span>")
+												"These words keep echoing in your mind. You find yourself completely fascinated by them.")]"))
+	to_chat(owner, span_danger("You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so,\
```
Comment (MrMelbert) [inline code/datums/brain_damage/hypnosis.dm:34]: boldwarning is a span, why is this changed to danger?

File: code/modules/hydroponics/unique_plant_genes.dm [hunk 4]
```diff
@@ -176,10 +176,10 @@
 
 	var/obj/item/seeds/our_seed = our_plant.get_plant_seed()
 	if(!our_seed.get_gene(/datum/plant_gene/trait/sticky) && prob(66))
-		to_chat(user, "<span class='danger'>[our_plant]'s thorns nearly prick your hand. Best be careful.</span>")
+		to_chat(user, span_userdanger("[our_plant]'s thorns nearly prick your hand. Best be careful."))
 		return
 
-	to_chat(user, "<span class='danger'>[our_plant]'s thorns prick your hand. Ouch.</span>")
+	to_chat(user, span_userdanger("[our_plant]'s thorns prick your hand. Ouch."))
```
Comment (MrMelbert) [inline code/modules/hydroponics/unique_plant_genes.dm:182]: These shouldn't be userdanger, just danger

**Review by MrMelbert — COMMENTED** [review]
I noticed a few spans definitely weren't translated over accurately

File: code/datums/brain_damage/magic.dm [hunk 5]
```diff
@@ -25,7 +25,7 @@
 		return
 
 	if(COOLDOWN_FINISHED(src, damage_warning_cooldown))
-		to_chat(owner, span_warning("<b>The light burns you!</b>"))
+		to_chat(owner, span_userdanger("The light burns you!"))
```
Comment (MrMelbert) [inline code/datums/brain_damage/magic.dm:28]: This was also undocumented upped to userdanger but I don't have a problem with it tbh

File: code/datums/brain_damage/hypnosis.dm (hunk 1 above)
Comment (jlsnow301) [reply code/datums/brain_damage/hypnosis.dm:28]: I was wondering about this. Perhaps I'll make a hypnophraselarge helper for it

File: code/datums/brain_damage/magic.dm (hunk 5 above)
Comment (jlsnow301) [reply code/datums/brain_damage/magic.dm:28]: I'm trying to keep some consistency with the context of spans. Warning is more of a "you tried this and it didn't work", danger is "you could be injured"

Comment (MrMelbert) [reply code/datums/brain_damage/magic.dm:28]: If you're updating spans in general (ie, changing them to new spans) you should mention that (I thought this was just straight conversions pr)

File: code/datums/elements/digitalcamo.dm [hunk 6]
```diff
@@ -39,10 +39,10 @@
 		var/datum/atom_hud/S = GLOB.huds[AI.sec_hud]
 		S.unhide_single_atomhud_from(AI,target)
 
-/datum/element/digitalcamo/proc/on_examine(datum/source, mob/M)
+/datum/element/digitalcamo/proc/on_examine(datum/source, mob/viewer)
 	SIGNAL_HANDLER
 
-	to_chat(M, "<span class = 'warning'>[source.p_their()] skin seems to be shifting and morphing like is moving around below it.</span>")
```
Comment (jlsnow301) [inline code/datums/elements/digitalcamo.dm:45]: hmm??

File: code/modules/projectiles/guns/ballistic/bow.dm [hunk 7]
```diff
@@ -46,7 +46,7 @@
 	if(!chambered)
 		return
 	if(!drawn)
-		to_chat(user, "<span clasas='warning'>Without drawing the bow, the arrow uselessly falls to the ground.</span>")
```
Comment (jlsnow301) [inline code/modules/projectiles/guns/ballistic/bow.dm:49]: wew

File: code/datums/brain_damage/magic.dm (hunk 5 above)
Comment (jlsnow301) [reply code/datums/brain_damage/magic.dm:28]: done - i've also tried to outline my thoughts on the changes for consistency.

**Comment by jlsnow301** [comment]
This is quickly spilling over with feature creep so I will make the corrections I wanted and atomize the conversions + additions elsewhere (perhaps when feature freeze is over)
